app/controllers/invitations_controller.rb

```diff
@@ -62,7 +62,7 @@ def invitation_params
   end
 
   def sign_up_params
-    params.require(:user).permit(:name, :surname, :gender, :email, :password, :password_confirmation)
```

Comment:
So why you don't take a course on git then? `Try git` or `Git branching` for example.